### DISCUSSION TITLE
installation instructions

### DISCUSSION THREAD

**Discussion by abubelinha** [JordyHers/Times-up-flutter#246] [Q&A]
Hi. this project looks great but I am a bit lost.
Any installation instructions available?

Is this something only for programmers, so we need to figure out how to compile and install ourselves?
Or is there an APK downloadable somewhere?

Thanks
@abubelinha

**Comment by JordyHers** [comment]
Hello there. The project is open source and still in development. I will suggest to join the community, you will find all the information there. Aside from that if you want an apk version to test locally I will upload one. You will not be able to sign up with Google but you will have it locally and test the demo version. 

Here is the link to join the community 
Discord

https://discord.gg/NBFyRmJd35

**Reply by mrjamesweb** [reply]
Hello
